.travis.yml

```diff
@@ -160,4 +160,5 @@ deploy:
       tags: true
       condition:
       - "$DEPLOY_DOCS == true"
+  - edge: true
```

Comment:
The test fails with a warning if you don't use the new API. However, they did not make it clear that you need to enable the new API.